app/utils/callback/lava.py

```diff
@@ -36,6 +36,8 @@
 import json
 import urllib2
 from collections import OrderedDict
+from urlparse import urlparse
```

Comment:
It's worth adding 2020, 2021 to the Collabora copyright statement.

Reply:
added